### PR TITLE
Add 'nng' package

### PR DESCRIPTION
**Pull request by tnixeu**
* I've followed [this guide](https://docs.hunter.sh/en/latest/creating-new/create/cmake.html)
  step by step carefully. **Yes**

* I've tested this package remotely and have excluded all broken builds.
  Here is the links to the Travis/AppVeyor with status "All passed": **[Yes]**
  
  * https://ci.appveyor.com/project/tnixeu/hunter-old/builds/28387878
  * https://travis-ci.org/tnixeu/hunter_old/builds/602955266

* I have submitted CI configs to https://github.com/cpp-pm/hunter-testing targeting `pkg.template` branch,
  see this merged pull request https://github.com/cpp-pm/hunter-testing/pull/9

### CODE REVIEW

**Comment by bkotzz** [comment]
Looks like the build failed for a few of the toolchains - can you comment those out in the hunter-testing repo?

File: docs/packages/pkg/nng.rst [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  single: unsorted ; nng
+
+.. _pkg.nng:
+
+nng
+===
+
+-  `Official <https://nanomsg.github.io/nng/index.html>`__
+-  `Example <https://github.com/cpp-pm/hunter/blob/master/examples/nng/CMakeLists.txt>`__
+-  Added by `tnixeu <https://github.com/tnixeu>`__ (`pr-N <https://github.com/cpp-pm/hunter/pull/N>`__)
```
Comment (bkotzz) [inline docs/packages/pkg/nng.rst]: I personally don’t see much value in this line in a world with git blame but officially this PR number should be filled in

Comment (tnixeu) [reply docs/packages/pkg/nng.rst]: Ok, i'll remove it.

**Comment by tnixeu** [comment]
I inserted the PR number and removed the developer name.

**Comment by rbsheth** [comment]
@tnixeu Can you put the developer name back? Let's strive for consistency amongst the docs, at least for now. Thanks.

**Comment by tnixeu** [comment]
I added the developer name again

**Comment by rbsheth** [comment]
Released: https://github.com/cpp-pm/hunter/releases/tag/v0.23.222

**Pull request merged**